native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -146,8 +155,11 @@ class BpkTextInput extends Component<Props, State> {
       valid,
       onFocus,
       onBlur,
+      accessoryView,
       ...rest
     } = this.props;
+    const hasAccessoryView = accessoryView != null;
```

Comment:
should use `!==`

Reply:
Yeah I think I had a thought here, but it's probably a dumb idea anyway